lib/rules/no-duplicate-selectors/index.js

```diff
@@ -62,7 +63,11 @@ const rule = (primary, secondaryOptions) => {
 			);
 			const resolvedSelectorList = [
 				...new Set(
-					ruleNode.selectors.flatMap((selector) => resolvedNestedSelector(selector, ruleNode)),
+					ruleNode.selectors.flatMap((selector) => {
```

Comment:
[trivial] In my opinion, `filter()` can make it a bit readable. But no problem as is. 👍🏼 

```js
ruleNode.selectors
	.filter(isStandardSyntaxSelector)
	.flatMap((selector) => resolvedNestedSelector(selector, ruleNode)),
```

Reply:
I would prefer current implementation because it does not need to iterate `selectors` twice.